README.md

```diff
@@ -271,18 +272,54 @@ snphost config reset
 
 ### 9. `verify`
 
-Reads the certificates in a directory and verifies the certificate chain, ensuring its integrity and authenticity. This command is essential for validating the trustworthiness of the certificates that can be then passed to complete attestation.
+Verify components that can be used in different SNP attestation workflows.
+
+**Usage:**
+```sh
+snphost verify <subcommand>
+```
+
+**Subcommands:**
+
+#### 1. `certs`
+Verifies a certificate chain to make sure the correct signatures are applied to each certificate in the chain.
 
 **Usage:**
 ```bash
-snphost verify DIR-PATH
+snphost verify certs DIR-PATH
 ```
 
 **Example:**
 ```bash
 snphost verify ./certs
 ```
 
+#### 2.  `vlek-hashtick`
+Verifies thet the TCB values in a vlek-hashstick are valid. This let's a user know that the provided hashstick is valid before trying to loaded in the system.
+
+**Usage:**
+```bash
+snphost verify vlek-hashtick HASHSTICK-PATH
+```
+
+**Example:**
+```bash
+snphost verify vlek-hashtick hashstick.bin
+```
+
+### 10. `vlek-load`
+Loads a vlek-hashstick into the system so that users can do attestation with a valid VLEK.
```

Comment:
```suggestion
Loads a vlek-hashstick binary into the system so that users can do attestation with a valid VLEK.
```
This is to clarify that the raw response from KDS `/hash_sticks` API cannot be used directly